hal/inc/hal_platform_compat.h

```diff
@@ -108,4 +108,8 @@
 #define PRODUCT_SERIES                      "electron"
 #endif
 
+#if PLATFORM_ID == PLATFORM_BSOM
+#define HAL_PLATFORM_MUXER_MAY_NEED_DELAY_IN_TX (1)
+#endif // PLATFORM_ID == PLATFORM_BSOM
```

Comment:
@avtolstoy Please verify if this is the correct place to set for `PLATFORM_BSOM`